manta-pay/Cargo.toml

```diff
@@ -140,4 +140,6 @@ ws_stream_wasm = { version = "0.7.3", optional = true, default-features = false
 [dev-dependencies]
 manta-crypto = { path = "../manta-crypto", default-features = false, features = ["getrandom"] }
 manta-pay = { path = ".", default-features = false, features = ["groth16", "scale", "scale-std", "std", "test"] }
+manta-sdk = { git = "https://github.com/manta-network/sdk.git", default-features = false, features = ["download"]}
```

Comment:
This needs to anchor a specific `manta-sdk` version. (cc @bhgomes )

Reply:
@bhgomes which `manta-sdk` version shall we use?

Reply:
We're going to have to think about this. We might have to move the "concrete parameters" out of SDK. I'll meditate on this over the weekend.